internals/router/swagger_custom_middleware.go

```diff
@@ -47,10 +47,14 @@ func modifySwaggerHTML(html string) string {
 		customHeader.style.fontWeight = '700';
 		customHeader.style.fontFamily = 'Titillium Web,sans-serif';
 		customHeader.style.color = '#ffffff';
-		customHeader.style.backgroundColor = color;
-		customHeader.textContent = title;
+		if (color != '') {
+			customHeader.style.backgroundColor = color;
+		}
+		if (title != '') {
+			customHeader.textContent = title;
```

Comment:
If the two are blank "", dont even modify the header !
	topbarColor := viper.GetString("SWAGGER_TOPBAR_COLOR")
	topbarTitle := viper.GetString("SWAGGER_TOPBAR_TITLE")

if topcolor != "" && topbartitle != "" {
				modifiedHTML := modifySwaggerHTML(rec.body.String(), topcolor, topbartitle)
				w.Header().Set("Content-Type", "text/html")
				w.Write([]byte(modifiedHTML))
}

useless to write to header if the settings are empty.